ir.c

```diff
@@ -1600,6 +1603,9 @@ int ir_mem_flush(void *ptr, size_t size)
 #if ((defined(__GNUC__) && ZEND_GCC_VERSION >= 4003) || __has_builtin(__builtin___clear_cache))
 	__builtin___clear_cache((char*)(ptr), (char*)(ptr) + size);
 #endif
+#if defined(__APPLE__) && defined(__aarch64__)
+	__clear_cache((char*)(ptr), (char*)(ptr) + size);
```

Comment:
Are you sure, if ``__clear_cache()`` is defined in `` <libkern/OSCacheControl.h>``?
I guess, ``sys_icache_invalidate()`` should be used instead.